certbot/cli.py

```diff
@@ -620,9 +620,9 @@ def parse_args(self):
                                    " wildcard domain is not supported.")
 
 
-        if self.verb == "register" and parsed_args.update_registration:
-            sys.stderr.write("Usage 'certbot register --update_registration' is deprecated.\n"
-                           "Use 'cerbot update_registration [options]' instead.\n")
+        if self.verb == "register" and parsed_args.update_account:
```

Comment:
I'm sorry I didn't think of this when you asked where we should rename `registration` to `account`, but since the name of the flag is `--update-registration` I think we should still use `update_registration` here as well to make it clearer this variable referring to the old deprecated flag.